.github/CONTRIBUTION.md

```diff
@@ -64,6 +64,12 @@ You can skip git commit hooks with:
 
     git commit -m "commit message" --no-verify
 
+## Deduplicate packages
+
+Yarn v1 does not support package deduplication natively. yarn-deduplicate is used to cleans up yarn.lock by removing duplicates:
+
+      npx yarn-deduplicate yarn.lock
+
```

Comment:
This feels a bit out of context. All the other docs here explain normal things you do when developing. This is more of a maintenance thing. And pretty generic, it just describes what `yarn-deduplicat` does. I think if we want it it in here, then we should at least have a command for it, if that is possible

Reply:
`npx yarn-deduplicate yarn.lock` is the command.

Reply:
ah you changed it, nice